moja-global/FLINT-UI
# Consistent margin for division around header

## Description

Fixes #191 
Added and removed margins and paddings to make header consistent across all pages.


## Testing

1. Open Flint.UI home page
2. Notice the header.
3. Go to Point /RothC / GCBM page.
4. Notice the alignment 


## Additional Context (Please include any Screenshots/gifs if relevant)
![image](https://user-images.githubusercontent.com/55620053/159138324-70e8536b-2256-486e-ba60-832c843e200e.png)

![image](https://user-images.githubusercontent.com/55620053/159138333-a92da6b8-41d3-41e3-bae9-83e6af4f1232.png)
